README.md

```diff
@@ -1,11 +1,26 @@
 # Melange
 
-A _mixture_ of tooling combined to produce JS from OCaml / Reason.
+### Tools for producing JS from Reason/OCaml.
 
-This project is a fork of the
-[ReScript compiler](https://github.com/rescript-lang/rescript-compiler/) with
-a focus on compatibility with the wider OCaml ecosystem. A small write-up with
-more details on the motivation behind this project can be found in this
+Melange is a compiler for producing performant Javascript from Reason and Ocaml.
```

Comment:
This is not 100% accurate. `melc` is the compiler, but we ship `mel` too, which produces dune rules (and invokes dune to build)

Reply:
Could you use the correct capitalization for JavaScript, according to the trademark, please? https://www.oxfordreference.com/view/10.1093/acref/9780199571444.001.0001/acref-9780199571444-e-1823